### PR TITLE
PIM-6814: Fix job profile creation validation

### PR DESCRIPTION
**Pull request by phaseinducer**
**Description (for Contributor and Core Developer)**

When you tried to create a job profile, it was failing because of EE permissions, see EE PR for more details.

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added Behats                      | OK
| Added integration tests           | -
| Changelog updated                 | OK
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | OK
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

**Comment by jjanvier** [comment]
about the saver, imho, we should not merge that

I'd be really happy to understand what is your use case behind this

**Comment by phaseinducer** [comment]
Discussed IRL with @jjanvier, it's ok. See the EE PR for the use case.

File: CHANGELOG-2.0.md [hunk 1]
```diff
@@ -24,6 +24,15 @@
 - Change constructor of `Pim\Bundle\EnrichBundle\Controller\ProductController` to add `Oro\Bundle\SecurityBundle\SecurityFacade`, an acl and a template
```
Comment (pierallard) [inline CHANGELOG-2.0.md]: Double line

File: src/Pim/Bundle/EnrichBundle/Controller/Rest/AttributeGroupController.php [hunk 2]
```diff
@@ -84,6 +93,8 @@ class AttributeGroupController
      * @param SaverInterface                $attributeSaver
      * @param SecurityFacade                $securityFacade
      * @param SimpleFactoryInterface        $attributeGroupFactory
+     * @param EventDispatcherInterface      $eventDispatcher
+     * @param CollectionFilterInterface     $inputFilter
```
Comment (pierallard) [inline src/Pim/Bundle/EnrichBundle/Controller/Rest/AttributeGroupController.php]: Shouldn't we set = null for these new parameters ?

File: src/Pim/Bundle/EnrichBundle/Event/CategoryEvents.php [hunk 3]
```diff
@@ -47,7 +47,7 @@
      * The event listener receives an
      * Symfony\Component\EventDispatcher\GenericEvent instance.
      *
-     * @staticvar string
+     * @deprecated
```
Comment (pierallard) [inline src/Pim/Bundle/EnrichBundle/Event/CategoryEvents.php]: Add "Will be removed in 2.1, please use xxxx instead" please for the next tag

File: src/Pim/Bundle/EnrichBundle/Controller/Rest/AttributeGroupController.php (hunk 2 above)
Comment (phaseinducer) [reply src/Pim/Bundle/EnrichBundle/Controller/Rest/AttributeGroupController.php]: Seen with @nidup, not a high risk of bc break.

File: src/Pim/Bundle/EnrichBundle/Event/CategoryEvents.php (hunk 3 above)
Comment (phaseinducer) [reply src/Pim/Bundle/EnrichBundle/Event/CategoryEvents.php]: There's no replacement for the moment, we'll add more business events later but I don't know when and what.

File: CHANGELOG-2.0.md (hunk 1 above)
Comment (phaseinducer) [reply CHANGELOG-2.0.md]: removed

**Pull request merged**